website/docs/r/guardduty_detector.html.markdown

```diff
@@ -40,13 +43,20 @@ The following arguments are supported:
 The `datasources` block supports the following:
 
 * `s3_logs` - (Optional) Describes whether S3 data event logs are enabled as a data source. See [S3 Logs](#s3-logs) below for more details.
+* `kubernetes_audit_logs` - (Optional) Describes whether Kubernetes audit log events are enabled as a data source.  See [Kubernetes Audit Logs](#kubernetes-audit-logs) below for more details.
 
 ### S3 Logs
 
 This `s3_logs` block supports the following:
 
 * `enable` - (Required) If true, enables [S3 Protection](https://docs.aws.amazon.com/guardduty/latest/ug/s3_detection.html). Defaults to `true`.
 
+### Kubernetes Audit Logs
+
+This `kubernetes_audit_logs` block supports the following:
+
+* `enable` - (Required) If true, enables [Kubernetes Protection](https://docs.aws.amazon.com/guardduty/latest/ug/kubernetes-protection.html). Defaults to `true`.
```

Comment:
Does this default to `true` because thats what happens with `s3_logs` or does this default to `true` because thats what AWS default for guard duty is?

I ask because the aws default is changing to false relatively soon (this month, I believe).

Reply:
Frankly that's a great point.  I wrote the code just before they put that notice out.
I don't mind which way it works and I'm quite open to opinions.

Reply:
I am in no way a contributor here, but I am looking forward to this contribution so thanks for putting it in!
As for the default, IMO it should follow AWS's default. I think it just makes the cognitive load lower if the default in the provider matches the defaults of what AWS has in place.

Reply:
IMHO it's still time to do it right for Kubernetes datasource and so set to false by default as AWS decided to do.

In terms of costs it is a better decision to set it to false by default too. ;)

Reply:
As already shared, AWS decided on the 02/08 that GuardDuty for EKS Protection will remain off by default : https://aws.amazon.com/about-aws/whats-new/2022/01/amazon-guardduty-elastic-kubernetes-service-clusters/

After about 20 days of trial on a bunch of AWS accounts I see the bills rising rapidly and honestly given the different security means I already use on clusters (Policies), I am not totally convinced of the gain. I think that enabling EKS Protection in Guard Duty should be a conscious decision.

@quintok please reconsider setting the default to false.